treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -60,7 +60,7 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity):
 
     with transaction.atomic():
         for datum in series:
-            PerformanceDatum.objects.create(
+            PerformanceDatum.objects.get_or_create(
```

Comment:
Per job, we should only have one value per datum, so that behaviour is intentional. :)

However, I like the idea of just doing try...except here. I'll do that.